packages/godwoken/src/types.ts

```diff
@@ -51,10 +51,11 @@ export interface WithdrawalRequest {
   signature: HexString;
 }
 export interface WithdrawalLockArgs {
-  // the original deposition info
+  // the original deposit info
   // used for helping programs generate reverted custodian cell
-  deposition_block_hash: Hash;
-  deposition_block_number: HexNumber;
+  // deposition_block_hash: Hash;
+  // deposition_block_number: HexNumber;
```

Comment:
Should be deposit_block_hash ?

Reply:
Deleted, it's old version's info, now replaced with `account_script_hash`